src/layout_features.rs

```diff
@@ -0,0 +1,72 @@
+use allsorts::binary::read::ReadScope;
+use allsorts::font::Font;
+use allsorts::font_data::FontData;
+use allsorts::layout::{LangSys, LayoutTable};
+use allsorts::tag::DisplayTag;
+
+use crate::cli::LayoutFeaturesOpts;
+use crate::BoxError;
+
+pub fn main(opts: LayoutFeaturesOpts) -> Result<i32, BoxError> {
+    let buffer = std::fs::read(&opts.font)?;
+    let scope = ReadScope::new(&buffer);
+    let font_file = scope.read::<FontData>()?;
+    let provider = font_file.table_provider(0)?;
```

Comment:
Feel free to skip but you might like to add an `index` option to the CLI to allow the font index to be supplied ([like dump](https://github.com/yeslogic/allsorts-tools/blob/259b13e9b228c177b57f266c162eda8aa488739f/src/cli.rs#L98-L103))